packages/elements/src/pagination/index.ts

```diff
@@ -713,12 +703,13 @@ export class Pagination extends BasicElement {
           part="input"
```

Comment:
As discussed, change `status` role as per my previous comments.

Reply:
Do you mean to change the `spinbutton` role to status `role` on the part input?

Reply:
no, re:

- To start with you need either aria-labelledby which links with the label, and yes, the screen reader will pronounce label if used with aria-label on Tab/Enter (at least mine does).
- Second option you do not use aria-labelledby and use role=status with aria-live instead. In that scenario the label will be pronounced as soon as aria-label changes. To make it working correctly you may need to bind it with the focused state of input.
- But when looking at your scenario, you can just use aria-description and do not add status element at all.
- another option is to use aria-valuetext on the input itself, but I do not think it works with all screen readers

Reply:
**Let me separate this discussion into two issues.**
1. How to set the `label` for the `input` correctly.
2. How to make screen readers pronounce when submitting page value by pressing `Tab/Enter`.

**Solutions**
1. I found a new solution for the first issue by using a `label` element that has `for` attribute same as native. I will update the code for review.
2. For the second issue, I think we must have the `status` element to solve that, if we do not have it that is not good for the screen reader users because it should have a confirmation message after pressing `Tab/Enter` keys.

Reply:
I have already updated code, please review again.

Reply:
can you please explain, what do you try to achieve with status and why by just adding `aria-label` on input you cannot satisfy the requirement?

Reply:
Imagine, when a blind person uses a screen reader when they open a web browser and press `Tab` to focus on the `ef-pagination`, and then the screen reader will get the `label` of `ef-pagination` and pronounce it.  

This is normal related behavior of `label` and `input element` but this is not enough for `ef-pagination` why?

When using `ef-pagination` with a screen reader, the user (blind person) can press the arrow up/down, or type to update the value of the input, but these actions do not update the `page` of `ef-pagination` until pressed the `Enter/Tab` key.

The question is what will happen with the screen reader if the user (blind person) types a `new page number` or uses the `arrow up/down` key and presses `Enter`? The answer is nothing happened.

This is unfriendly for blind users because they will not know what will happen after submitting the change and they will not know if the page has already changed or not, which makes them confused.

That is the reason why `aria-label` is not enough and we should use the `role=status` with `aria-live` to make `ef-pagination` pronounce the `page number` after the page is already changed from submitted, this is the way it should be.

Reply:
If you have any question please let me know, I will call you.

Reply:
I disagree. I have set the call for tomorrow.

Reply:
<img width="997" alt="image" src="https://user-images.githubusercontent.com/87511654/160770077-cae9fac3-3c03-4996-ad1a-f885cdfb0b23.png">

@goremikins 
According to the meeting, we already decided to go with `Option 4` (In the image above) but I have already checked on all browsers, to make it works it must add more logic to the `ef-pagination` that might not be good because it is more complex.

@wsuwt and I think the same thing about `Option 1` it could be the best solution because it is simple and we do not add any logic to handle the `A11Y`, for the only one issue on `JAWS` (pronounce after pressing an `Enter` key) I think it is acceptable.

I have already pushed both options to the PR, please review them and let me know which one you prefer. 
Thanks, 😀

Option 4: https://github.com/Refinitiv/refinitiv-ui/pull/256/commits/79725c11617e023ce2a431ef5fcc1079a0963f2d
Option 1: https://github.com/Refinitiv/refinitiv-ui/pull/256/commits/2282fa80f240fc11da50e8e2a85a5bb1af17256d

Reply:
Let's go with option 1 than

Reply:
Please add the new re-usable mixin in elemental-theme, called ariaInvisible, which should have the following css:
```
clip: rect(1px, 1px, 1px, 1px); // IE11
clip-path: inset(50%);
height: 1px;
width: 1px;
margin: -1px;
overflow: hidden;
padding: 0;
position: absolute;
```
This block is derived from Bootstrap and was suggest by Caleb to hide content.

Reply:
@goremikins I have already added the mixing to the shared style, please review it again.